### DISCUSSION TITLE
DietPi-Software | Grav

### DISCUSSION THREAD

**Discussion by mews-se** [MichaIng/DietPi#6201] [Software Request]
#### Formal software information
- Software title | Grav
- Short description | Easy to use, yet powerful, opensource CMS that requires no database
- Official URL (if available) | https://getgrav.org/
- GitHub URL (if available) | https://github.com/getgrav/grav
- (Official) install documentation | https://learn.getgrav.org/

#### Are there similar/alternative software titles available with DietPi-Software?
- Wordpress

#### What makes your requested software better than the above solutions, if available?
- Smaller footprint and more alternatives for CMS

**Comment by MichaIng** [comment]
Converted to discussion, so others please vote for this software implementation via ⬆️ button at first post.
